templates/image-streams-ppc64le.json

```diff
@@ -324,6 +324,63 @@
         {
             "kind": "ImageStream",
             "apiVersion": "image.openshift.io/v1",
+            "metadata": {
+                "name": "ubi8-openjdk-17",
+                "annotations": {
+                    "openshift.io/display-name": "Red Hat OpenJDK 17 (UBI8)",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "1.4.17"
+                },
+                "labels": {
+                    "xpaas": "1.4.17"
+                }
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "1.10",
+                        "annotations": {
+                            "openshift.io/display-name": "Red Hat OpenJDK 17 (UBI8)",
+                            "description": "Build and run Java applications using Maven and OpenJDK 17 upon UBI8.",
+                            "iconClass": "icon-rh-openjdk",
+                            "tags": "builder,java,openjdk,ubi8,hidden",
+                            "sampleRepo": "https://github.com/jboss-openshift/openshift-quickstarts",
+                            "sampleContextDir": "undertow-servlet",
+                            "version": "1.10"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.access.redhat.com/ubi8/openjdk-17:1.10"
+                        }
+                    },
+                    {
```

Comment:
this section (lines 359-377) is a duplicate of the previous item and should be removed